keps/sig-storage/3751-volume-attributes-class/README.md

```diff
@@ -402,10 +401,44 @@ Note:
 
 The parameters in VolumeAttributesClass are opaque and immutable. This gives different cloud providers flexibility but at the same time it is up to the specific driver to verify the values set in the parameters. The parameters from VolumeAttributesClass associated with a volume are mutable because they are coming from different VolumeAttributesClasses.
 
+**Deal with Volume Reverted to nil VAC**
+
+After reverting VAC name to nil, the volume may not be fully reverted to the previous state, because we only persistent the result of the last modify attempt, and not sure about whether the volume is changed in the previous attempts.
+We also don't know the previous state of the volume, so we cannot perform rollback.
+The PVC status will be left untouched to remind user and admin about this protential inconsistency. Now:
+* If there is no quota, user may not care about this, and just leave it there
+* User can set a (potentially different) VAC name to try again
+* Admin can check the volume status manually and reset PVC.status.modifyStatus to nil to fully revert the change.
+
+This may also confuse the higher level controller (e.g. [KEP-4650](https://github.com/kubernetes/enhancements/issues/4650)) when reconciling PVC. Higher level controllers can also special casing nil VAC names to skip waiting for the modification.
+
 ### Risks and Mitigations
 
+**Parameter Conflicts**
+
 Users may configure both parameters in StorageClass and parameters in VolumeAttributesClass differently. The mitigation here is to provide a guideline of solving conflicts. Drivers should return an error if they are conflicting. If the MODIFY_VOLUME capability is present, VolumeAttributesClass should be honored.
 
+**Quota Abuse**
+
+For administrators who want to set up quota on the VACs to control the expense, he should set up each VAC to fully specify the state of the volume. That is to say, each VAC should fully overwrite changes to every other VAC.
+
+Consider this bad setup that violates the previous requirement.
+* VAC A: `{"bandwidth": "100Mbps"}`
```

Comment:
It just sounds like to specify VAC parameters, it is better to include all modifiable parameters if the admin cares about quota.